R/utils-xportr.R

```diff
@@ -215,7 +215,7 @@ xpt_validate <- function(data) {
   }
 
   # 3.0 VARIABLE TYPES ----
-  types <- tolower(extract_attr(data, attr = "SAStype"))
+  types <- tolower(extract_attr(data, attr = "type"))
```

Comment:
The `xport_type()` function does not add any attributes, so I suggest to remove the type check.